pkg/adapter/adapter.go

```diff
@@ -65,6 +67,19 @@ func NewLocalAdapter(cfg *config.Config) (*LocalAdapter, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "create content provider")
 	}
+	ticker := time.NewTicker(time.Hour)
+	threshold, err := humanize.ParseBytes(cfg.Provider.GCPolicy.Threshold)
+	if err != nil {
+		return nil, err
+	}
+	// start scheduled gc task every hour
+	go func() {
+		for range ticker.C {
+			if err := content.GC(namespaces.WithNamespace(context.Background(), "acceleration-service"), (int64)(threshold/2)); err != nil {
+				logrus.Error(errors.Wrap(err, "scheduled gc task"))
+			}
+		}
+	}()
```

Comment:
Maybe we should put these codes into a separate func, then keep the NewLocalAdapter clearer.